iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -1071,6 +1071,16 @@ public void postGetTable(org.apache.hadoop.hive.metastore.api.Table hmsTable) {
         if (!"1".equals(formatVersion)) {
           hmsTable.getParameters().put(TableProperties.FORMAT_VERSION, formatVersion);
         }
+        // Set the serde info
+        hmsTable.getSd().setInputFormat("org.apache.iceberg.mr.hive.HiveIcebergInputFormat");
```

Comment:
nit:  Can we replace the strings with `getClass().getName()` ?

Reply:
Done